src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,308 @@
+import React, { useEffect } from 'react';
+import classNames from 'classnames';
+import { NavLink, useParams } from 'react-router-dom';
+import { client } from '../../utils/fetchClient';
+import { Person } from '../../types';
+import { Loader } from '../Loader';
+import { PeopleFilters } from '../PeopleFilters';
+import { useTableContext } from '../Context';
+import { SearchLink } from '../SearchLink/SearchLink';
+
+export const PeoplePage: React.FC = () => {
+  const {
+    peoples,
+    setPeoples,
+    hasError,
+    setHasError,
+    isLoading,
+    setIsLoading,
+    currPeoples,
+    searchParams,
+    query,
+    gender,
+    centuries,
+    setCurrPeoples,
+    sortCurr,
+    order,
+  } = useTableContext();
+
+  const necessaryPeople = currPeoples ?? peoples;
+
+  const { slug } = useParams();
+
+  const sortParams = (sortType: string) => {
+    if (sortCurr !== sortType) {
+      return { sort: sortType, order: null };
+    }
+
+    if (order) {
+      return { sort: null, order: null };
+    }
+
+    return { sort: sortType, order: 'desc' };
+  };
+
+  useEffect(() => {
+    setIsLoading(true);
+
+    client.get('people.json')
+      .then((response: unknown) => {
+        setPeoples(response as Person[]);
+      })
+      .catch(() => {
+        setHasError(true);
+      })
+      .finally(() => {
+        setIsLoading(false);
+      });
+  }, []);
```

Comment:
fix this warning 
![image](https://github.com/mate-academy/react_people-table-advanced/assets/70009205/c5d6f6f6-b469-4506-b6e2-4956af098908)